src/include/homestore/btree/detail/btree_internal.hpp

```diff
@@ -254,35 +254,32 @@ struct BtreeConfig {
 
 public:
     BtreeConfig(uint32_t node_size, const std::string& btree_name = "") :
-            m_node_size{node_size},
-            m_node_data_size{node_size},
-            m_btree_name{btree_name.empty() ? std::string("btree") : btree_name} {
-        m_ideal_fill_size = (uint32_t)(m_node_data_size * m_ideal_fill_pct) / 100;
-        m_suggested_min_size = (uint32_t)(m_node_data_size * m_suggested_min_pct) / 100;
+            m_node_size{node_size}, m_btree_name{btree_name.empty() ? std::string("btree") : btree_name} {
```

Comment:
Generally all btree/ directory changes are cosmetic in nature and can be skipped review if need be.